dimagi/pages/templates/amp/base.amp.html

```diff
@@ -0,0 +1,247 @@
+{% load i18n %}{% load dimagidotcom %}{% load capture_tags %}<!DOCTYPE html>
+{# note: modernizr.js and app.js add classes to <html> to trigger certain css behaviors. #}
+{# These classes are: #}
+{#     js, no-js, touch, no-touch, cssanimations, #}
+{#     csstransformations3d, csstransitions, video, audio, svg, inlinesvg #}
+{#     pointerevents, show-time #}
+<html amp lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width,minimum-scale=1, initial-scale=1, user-scalable=no, shrink-to-fit=no, viewport-fit=cover">
+    <meta name="format-detection" content="telephone=no">
+    <meta property="dimagi:baseUrl" content="{{ meta.baseUrl }}">
+
+    <meta property="dimagi:trackingConfig" content="{{ meta.trackingConfig|JSON|META_STR }}">
+    <meta property="dimagi:tracking" content="{{ meta.tracking|JSON|META_STR }}">
+    <meta property="dimagi:trackingAB" content="{{ meta.trackingAB|JSON|META_STR }}">
+
+    <meta property="dimagi:auditABClicks" content="{{ meta.auditABClicks|BOOL }}">
+
+    <meta property="dimagi:contactFormId" content="{% block contact_form_id %}160467c5-1ae9-401c-81f2-09334ce10b1a{% endblock %}">
+    <meta property="dimagi:demoFormId" content="{% block demo_form_id %}afb4d5a0-826f-4a56-8c21-552d4b6dafa3{% endblock %}">
+
+    {% if request.hide_drift %}
+      <meta property="dimagi:driftHide" content="true">
+    {% endif %}
+
+    {% capture as page_title silent %}
+      {% block title %}Dimagi{% endblock %}
+    {% endcapture %}
+
+    {% capture as page_description silent %}{% block description %}
+      {% blocktrans %}
+        Dimagi builds technology solutions designed for impact. We on a mission
+        to improve the efficiency and quality of impact-driven work globally.
+        Our core product is CommCare - mobile data collection with the power
+        of case management.
+      {% endblocktrans %}
+    {% endblock %}{% endcapture %}
+
+    {% capture as page_social_card silent %}{% block social_card_image_url %}
+      {% static 'pages/home/default-social-card.jpg' %}
+    {% endblock %}{% endcapture %}
+
+    <title>{{ page_title|escape|TRIM }}</title>
+    <meta name="description"
+          content="{{ page_description|escape|TRIM }}">
+
+    <!-- Twitter Card data -->
+    <meta name="twitter:card" content="summary_large_image">
```

Comment:
i have the same comment on meta tags as above. please pull these into partials to share between the templates

Reply:
actually, the better thing would be to keep the same `base.html` template and use a server-side AMP flag determined by the User Agent to hide / show the amp content changes on the original `base.html` rather than creating a whole separate base template, as there is too much duplicate code otherwise.